.dockerignore

```
@@ -3,3 +3,4 @@
 .settings/
 */target/
 dspace/modules/*/target/
+Dockerfile.*
```

```suggestion
Dockerfile.*

```
is there a preference to add a new line at the end of a file?

---

There is no preference either way.